diff --git a/src/Plugin/CKEditorPlugin/CodeSnippet.php b/src/Plugin/CKEditorPlugin/CodeSnippet.php
index 1f95556..a292bdd 100644
--- a/src/Plugin/CKEditorPlugin/CodeSnippet.php
+++ b/src/Plugin/CKEditorPlugin/CodeSnippet.php
@@ -27,7 +27,7 @@ class CodeSnippet extends CKEditorPluginBase implements CKEditorPluginConfigurab
    * {@inheritdoc}
    */
   public function getFile() {
-    return base_path() . 'libraries/codesnippet/plugin.js';
+    return 'libraries/codesnippet/plugin.js';
   }
 
   /**
